### PR TITLE
ProtoBufs :parse  token program and token extensions accounts  

### PR DESCRIPTION
**Pull request by Nagaprasadvr**
- [ ] Add Token Program Protobufs
- [ ] Add Token Extensions Protobufs

### CODE REVIEW

File: crates/parser/src/helpers.rs [hunk 1]
```diff
@@ -58,3 +59,20 @@ impl<A, D> ReadableInstruction<A, D> {
...
+
+pub trait IntoProtoData<O> {
+    fn into_proto_data(self) -> O;
+}
+
+pub fn pubkey_to_string(pubkey: SolanaPubkey) -> String { pubkey.to_string() }
+
+pub fn bytes_to_elgamal_pubkey(bytes: [u8; ELGAMAL_KEYPAIR_LEN / 2]) -> String {
+    bs58::encode(bytes).into_string()
+}
```
Comment (kespinola) [inline crates/parser/src/helpers.rs]: Since L63, 74, 76 are type conversion represent them using the From trait.

**Review by kespinola — CHANGES_REQUESTED** [review]

**Comment by kespinola** [comment]
@Nagaprasadvr can you add a README.me to the streams examples about to listen using grpcurl.

File: crates/parser/src/helpers.rs (hunk 1 above)
Comment (Nagaprasadvr) [reply crates/parser/src/helpers.rs]: these types are from external crate so i guess we cant impl From Trait

**Review by ray-kast — APPROVED** [review]

File: crates/parser/src/helpers.rs (hunk 1 above)
Comment (kespinola) [reply crates/parser/src/helpers.rs]: SolanaPubkey(pubkey::Pubkey)
ElgamalPubkeyBytes([u8; ELGAMAL_KEYPAIR_LEN / 2])

and then define from traits on these structs.

Comment (Nagaprasadvr) [reply crates/parser/src/helpers.rs]: fixed

**Review by kespinola — APPROVED** [review]

**Pull request merged**